src/_assets/css/_bootstrap_config.scss

```diff
@@ -27,7 +27,9 @@ $container-max-widths: (
   xxl: 1330px
 );
 $grid-breakpoints: (
-  xs: 0,
+  0: 0,
+  xxs: 320px,
+  xs: 480px,
```

Comment:
@fertolg - note the two new breakpoints, which are used to style mobile-screenshot images. This shouldn't affect anything else (related to breakpoints) on the site, but let's keep an eye open for that. (I'll look into using another approach if it does cause problems.)